### PR TITLE
Added compound literals

### PR DESCRIPTION
**Pull request by iherman**
I picked up #22 and added to the document as yet another, possible solution; thanks to @afs  

@afs can you look at this to see if it makes sense?

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-dir-literal/pull/23.html" title="Last updated on Aug 29, 2019, 1:16 PM UTC (975f6ae)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-dir-literal/23/8447257...975f6ae.html" title="Last updated on Aug 29, 2019, 1:16 PM UTC (975f6ae)">Diff</a>

### CODE REVIEW

**Comment by afs** [comment]
Looks good! Thanks @iherman

File: index.html [hunk 1]
```diff
@@ -413,11 +413,119 @@ <h2>General comments: Pros and Cons</h2>
...
+                    <p>
+                        The major advantage of this approach is that it is fully transparent to current RDF deployments, and affects only application layers that do display the literal values for humans. They also work for parsers “out of the box”. As an extra bonus (as mentioned <a href='https://github.com/w3c/EasierRDF/issues/22'>“Language Tagged Strings”</a> discussion) it may actually be easier to search such compound literals via, say, SPARQL then using the current RDF approach.
+                    </p>
+
+                    <p>
+                        The obvious disadvantage is that it duplicates a mechanism for expressing string with the language tag <em>only</em>: one can use the core <code>langString</code> datatype <em>or</em> the compound literals. The relationship between the two forms should be specified.
+                    </p>
+
+                    <p>
+                        (It would be possible to make a “mix” whereby, instead of using <code>rdf:language</code>, one would use a <code>rdf:langString</code> as an object for <code>rdf:value</code>.)
```
Comment (pchampin) [inline index.html]: Yuk :-/ I am not sure I want to hint at that solution... IMO, it has the worse of both worlds: there is still an indirection to follow to "find" the actual string, without the benefit of cleanly separating its metadata in distinct triples... Worth, the metadata are heterogeneous in that respect.

I suggest we remove this parenthesis.

**Review by pchampin — APPROVED** [review]
I like the text as it is (except for the parenthesis about mixing langString and compount literals, but that is a detail).

However, it would seem clearer to include it in the "RDF based solutions" section. After all, it *is* purely RDF based. The current solutions in this section leverage.

@iherman if you agree, I can make the change.

File: index.html (hunk 1 above)
Comment (iherman) [reply index.html]: I am not saying I like it... :-)

I am fine removing it, but you can bet someone will raise this!

**Comment by iherman** [comment]
I am fine moving it to the RDF section. I would appreciate you do it:-)

**Comment by pchampin** [comment]
>  I would appreciate you do it:-)

Done. Please have a look at the revise "General comments: Pros and cons" at the end of section 2.1 . I had to modify it slightly to account for the new RDF-based solution.

**Comment by iherman** [comment]
I changed only one thing: by adding this solution to the lot, it may not be _necessarily_ the case that a new WG is required to do this. Defining 3-4 extra terms may be done in another structure; e.g., I could imagine the I18N WG defining, formally, those terms. (Or may be even not requiring a WG but only a CG; to be discussed.)

I have somewhat weakened the last sentence, and I will now merge it. Discussions with this group to follow...

**Pull request merged**